components/03-modules/header/_search-form.scss

```diff
@@ -1,29 +1,37 @@
-$search-form__height               : 34px !default;
-$search-form__input-height         : 32px !default;
-$search-form__border-color         : $gray-dark !default;
-$search-form__border-color--xl     : $gray !default;
-$search-form__font-size            : $font-size-small !default;
-$search-form__placeholder-color    : $gray-dark !default;
-$search-form__placeholder-color--xl: $gray !default;
-$search-form__button-size          : 18px !default;
-$search-form__button-background    : transparent !default;
-$search-form__button-icon-color    : $gray-dark !default;
+$lavender-gray                 : #c2c1df;
+$kimberly                      : #73739c;
```

Comment:
it's already in global variables as grays colors :)

Reply:
Fixed.